src/swap/events.ts

```diff
@@ -6,6 +6,9 @@ export enum SwapEventName {
   SWAP_DETAILS_EXPANDED = 'Swap Details Expanded',
   SWAP_ERROR = 'Swap Error',
   SWAP_ESTIMATE_GAS_CALL_FAILED = 'Swap Estimate Gas Call Failed',
+  SWAP_FIRST_INPUT = 'Swap First Input',
```

Comment:
naming: Isn't this more like an action than an input? Thoughts on SWAP_FIRST_ACTION?